Assets/Scripts/Persona/api/APIClient.cs

```diff
@@ -33,12 +38,8 @@ public async Task<string> MakeGetRequestAsync(string url, Dictionary<string, str
             using (UnityWebRequest webRequest = UnityWebRequest.Get(url))
 
             {
-
                 headers ??= new Dictionary<string, string>();
 
-                headers.Add("x-api-key", PersonaAdSDK.GetApiKey());
```

Comment:
Is this not needed anymore ?

Reply:
yeah, x-api-key header is now already being added in BannerView.js file i.e. even before this point in code